src/tour/tour.js

```diff
@@ -194,6 +194,14 @@ angular.module('angular-tour.tour', [])
           scope.ttOffset = parseInt(val, 10) || tourConfig.offset;
```

Comment:
yeah, you're right. I'm starting to re-evaluate the backwards compatibility seeing as how `scope.ttOffset` is used for both vertical and horizontal offsets now. Perhaps its best to just axe `tourOffset` / `scope.ttOffset` entirely and just introduce the breaking change, since it's cleaner / more easily understandable.

I'd be fine if you did that, and updated the tests / demo app accordingly.